src/model/StraightComponent.hpp

```diff
@@ -54,16 +52,10 @@ namespace model {
     boost::optional<ModelObject> outletModelObject() const;
 
     /** Returns the optional AirLoopHVAC object that this AirToAirComponent is attached to.
-   *
-   *  Reimplemented from HVACComponent.
-   */
-    boost::optional<AirLoopHVAC> airLoopHVAC() const;
-
-    bool addToNode(Node& node);
-
-    ModelObject clone(Model model) const;
-
-    void disconnect();
+     *
+     *  Reimplemented from HVACComponent.
+     */
+    boost::optional<AirLoopHVAC> airLoopHVAC() const; // cppcheck-suppress [duplInheritedMember] for documentation purposes
```

Comment:
Some I keep, for documentation purposes mostly, because the docstring is helpful versus the base class one.